comdb2/factories.py

```diff
@@ -21,11 +21,16 @@
 A factory function will be called with a list of column names, and must return
 a callable that will be called once per row with a list of column values.
 """
+
+from __future__ import annotations
+
```

Comment:
```suggestion
```

Same

Reply:
Same as above, we wouldn't be able to use `list[Value]` if we did that, and would need to instead use `List[Value]`...